docs/troubleshooting-rbe.md

```diff
@@ -82,37 +82,37 @@ com.google.devtools.build.lib.remote.common.BulkTransferException: 3 errors duri
 
-The first step to verify this issue would be to copy the hash of the missing blob.
-Then navigate to the Invocation URL -> Cache -> "Cache requests" and paste the hash into the Filter input.
-This will let you know if Bazel has tried to upload the blob to BuildBuddy Remote Cache or not.
+The first step in verifying this issue is to copy the hash of the missing blob.
+Then, navigate to the Invocation URL -> Cache -> "Cache requests" and paste the hash into the Filter input.
+This will show whether Bazel has attempted to upload the blob to the BuildBuddy Remote Cache.
 
-If Bazel attempted to upload the blob and failed, there should be multiple retries attempted for the same blob.
-The retry attempts can be configured with the `--remote_retries` (default 5) and `--remote_retry_max_delay` (default 5s) flags.
-Additionally, `--experimental_collect_system_network_usage` (default true since Bazel 8) can be used to collect network usage data on Bazel's host machine.
+If Bazel attempted to upload the blob but failed, you should see multiple retry attempts for the same blob.
+The number of retry attempts and the delay between retries can be configured using the `--remote_retries` (default 5) and `--remote_retry_max_delay` (default 5s) flags.
+Additionally, `--experimental_collect_system_network_usage` (default true since Bazel 8) can be used to collect network usage data on the Bazel host machine.
 This network data will be displayed as a graph in the "Timing" tab of the Invocation page.
 
-If there was no attempt from Bazel to upload the missing blob, this is caused by a mismatch of expectation between Bazel's local state and the BuildBuddy Remote Cache.
-In a previous invocation (usually with Build without the Bytes turned on), Bazel local state was taught to assume that the blob is already in the Remote Cache.
-However, as time passed, the blob was evicted from BuildBuddy Remote Cache without Bazel's knowledge.
+If there was no attempt by Bazel to upload the missing blob, this signifies a mismatch between Bazel's local state and the BuildBuddy Remote Cache.
+In a previous invocation (often with Build without the Bytes enabled), Bazel's local state might have been configured to assume that the blob already exists in the Remote Cache.
+However, over time, the blob may have been evicted from the BuildBuddy Remote Cache without Bazel's knowledge.
 
-The best solution in this scenario is for Bazel to either re-upload the missing blob, or to re-execute the action that created the missing blob.
-This is also known as "Action Rewinding" in Bazel terminology.
-However due to the complexity of Bazel's code base, this feature is not yet fully implemented.
+The ideal solution is for Bazel to either re-upload the missing blob or re-execute the action that created it.
+This process is also known as "Action Rewinding" in Bazel.
+However, due to the complexity of Bazel's codebase, this feature isn't fully implemented yet.
```

Comment:
Is it enough to say "However, this feature isn't implemented yet." ?

Reply:
It's a good question. Technically there was a previous incarnation of this feature that is guarded behind this undocumented flag https://cs.opensource.google/bazel/bazel/+/master:src/main/java/com/google/devtools/build/lib/buildtool/BuildRequestOptions.java;l=348-354;drc=ee698448c8fe6f4e573e1fc1bbe14807c75b4c8f;bpv=0;bpt=1

I prefer to think of it as an experimental alpha version.

But I think what we want is a "newer" version that is more stable and backward compatible.

I guess we can just think of them as 2 separate features and call the one we want unavailable? cc: @fmeum 

```suggestion
However, this feature is not yet available in Bazel 8.
```